data-serving/scripts/convert-data/converters.py

```diff
@@ -374,30 +375,44 @@ def convert_notes_field(notes_fields: [str]) -> str:
     return notes or None
 
 
-def convert_sources_field(source: str) -> Dict[str, str]:
+def convert_case_reference_field(id: str, source: str) -> Dict[str, str]:
     '''
-    Converts the source field to a new source representation that's a list of
-    objects with either a URL or an unknown reference type.
+    Converts the case reference field from the source field.
 
     Returns:
       None: When the input is empty.
       Dict[str, str]: When the input is nonempty. The dictionary is in the
         format:
         {
-          'url': str,
-          'other': str
+          'sourceUrl': str,
```

Comment:
> Granted there's no real `sourceId` for this data, but it's a little odd that we're inserting data that disagrees with our Mongoose validation rules. Is it worth filling the ID with a hash of the URL (or perhaps some portion of it)?

I guess I was hoping we'd be able to backfill some of these ids, after we've added some data to the sources collection -- does that sound reasonable, and if reasonable, worth it?

> Mostly related: should we add a `required` stanza(s) in our json schema?

unfortunately, there are a few existing cases that do not have a source URL, so they would fail to validate & insert into the db :( I can look into getting those fixed upstream though and then we can make it required